keyboard.html

```diff
@@ -126,7 +126,9 @@ <h2 id="film" class="no-display">Video on Keyboard Compatibility</h2>
       <p>[ <strong>Previous:</strong> <em>none</em> | <a href="Overview.html">Up: Overview</a> | <a href="contrast.html"><strong>Next:</strong> Colors with Good Contrast</a> ]</p>
 
       <h2 id="what">What is Keyboard Compatibility?</h2>
-      <p>All functionality needs to work with the keyboard. That is, users can access and move between links, buttons, forms, and other controls using the Tab key and other shortcuts. Websites should not require a mouse; for example, pop-up calendars should also let users type in a date.</p>
+      <p>All functionality needs to work with the keyboard, so users can move between links, buttons, forms, and other controls. It is important that this works with standard keyboard interfaces.
+      Shortcuts for a website must not change the behaviour of the user's existing environment, e.g. overriding an important key, and users must be able to discover what shortcuts are available.  
```

Comment:
This seems a bit of a scope creep and too detailed for such a beginner resource. It was suggested by @hkramer to change the term "shortcut" to "keystrokes", which is clearer and possibly address your concern as well.